docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -0,0 +1,196 @@
+
+@sphinxdirective
+
+.. dropdown:: OpenCV
+
+   OpenCV is necessary to run demos from Open Model Zoo (OMZ). Some OpenVINO samples can also extend their capabilities when compiled with OpenCV as a dependency. The Intel® Distribution of OpenVINO™ provides a script to install OpenCV: ``<INSTALL_DIR>\extras\scripts\download_opencv.ps1``.
+
+   .. note::
+      No prerequisites are needed.
+      
+   There are three ways to run the script:
+   
+   * GUI: right-click the script and select ``Run with PowerShell``.
+      
+   * Command prompt (CMD) console:
+   
+   .. code-block:: sh
+   
+      powershell <INSTALL_DIR>\extras\scripts\download_opencv.ps1
+      
+      
+   * PowerShell console:
+   
+   .. code-block:: sh
+   
+      .\<INSTALL_DIR>\extras\scripts\download_opencv.ps1
+      
+
+   If the Intel® Distribution of OpenVINO™ is installed to the system location (e.g. ``Program Files (x86)``) then privilege elevation dialog will be shown. The script can be run from CMD/PowerShell Administrator console to avoid this dialog in case of system-wide installation. 
+   The script is interactive by default, so during the execution it will wait for user to press ``Enter`` If you want to avoid this, use the ``-batch`` option, e.g. ``powershell <INSTALL_DIR>\extras\scripts\download_opencv.ps1 -batch``. After the execution of the script, you will find OpenCV extracted to ``<INSTALL_DIR>/extras/opencv``.
+
+@endsphinxdirective
+
+### <a name="optional-steps"></a>Step 4 (Optional): Configure Inference on non-CPU Devices
+
+@sphinxdirective
+.. tab:: GNA
+
+   To enable the toolkit components to use Intel® Gaussian & Neural Accelerator (GNA) on your system, follow the steps in :ref:`GNA Setup Guide <gna guide windows>`.
+   
+.. tab:: GPU
+
+   To enable the toolkit components to use processor graphics (GPU) on your system, follow the steps in :ref:`GPU Setup Guide <gpu guide windows>`.
+
+.. tab:: VPU
+
+   To install and configure your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, see the :ref:`VPU Configuration Guide <vpu guide windows>`.
+   
+.. tab:: NCS 2
+   
+   No additional configurations are needed.
+   
+
+@endsphinxdirective
+
+## <a name="get-started"></a>What's Next?
+
+Now you are ready to try out the toolkit.
+
+Developing in Python:
+   * [Start with tensorflow models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/101-tensorflow-to-openvino-with-output.html)
+   * [Start with ONNX and PyTorch models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/102-pytorch-onnx-to-openvino-with-output.html)
+   * [Start with PaddlePaddle models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/103-paddle-onnx-to-openvino-classification-with-output.html)
```

Comment:
These notebooks will not work out of the box after following the steps in this guide. They require installing additional components and notebooks install instructions includes installation of openvino-dev (which includes openvino runtime).

Reply:
hmmm, maybe we should recommend the ones that can work in a browser with binder. What do you think?

Reply:
Binder is great to quickly get started, but performance and availability are limited so it would be good to also refer to the installation instructions for the notebooks as an optional step. Also because after installing you may want to know how to get things working locally (because you do not want to develop your own python scripts in Binder). But it is a bit unfortunate that at the moment notebooks installation is a separate process that requires many additional dependencies.  I agree we should not have that here as a mandatory step when you think you're done with installing. 

In the future we could have a way to get started with a subset of notebooks that does not require additional installation. The hello world notebooks (000 series) should work out of the box after this installation if someone has Jupyter installed.

Reply:
Updated.